modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -340,6 +350,45 @@ private String _getObjectEntryPermissionName(long objectDefinitionId) {
 		return ObjectDefinition.class.getName() + "#" + objectDefinitionId;
 	}
 
+	private void _hasPreventDeletionType(long objectDefinitionId) {
+		List<ObjectRelationship> objectRelationships =
+			_objectRelationshipLocalService.getObjectRelationships(
+				objectDefinitionId);
+
+		_hasPreventTypeRelationship = false;
+
+		for (ObjectRelationship objectRelationship : objectRelationships) {
+			if (StringUtil.equals(
+					objectRelationship.getDeletionType(),
+					ObjectRelationshipConstants.DELETION_TYPE_PREVENT)) {
+
+				_preventObjectRelationships.add(objectRelationship);
+				_hasPreventTypeRelationship = true;
+			}
+		}
+	}
+
+	private boolean _hasRelatedModels(
+			com.liferay.object.model.ObjectEntry objectEntry)
+		throws PortalException {
+
+		for (ObjectRelationship objectRelationship :
+				_preventObjectRelationships) {
+
+			int count =
+				_objectEntryLocalService.getOneToManyRelatedObjectEntriesCount(
+					objectEntry.getGroupId(),
+					objectRelationship.getObjectRelationshipId(),
+					objectEntry.getPrimaryKey());
+
+			if (count > 0) {
+				return true;
+			}
```

Comment:
In this case, can you include the method call inside of the `if` condition? For example:
```
if (_objectEntryLocalService.getOneToManyRelatedObjectEntriesCount(
		objectEntry.getGroupId(),
		objectRelationship.getObjectRelationshipId(),
		objectEntry.getPrimaryKey()) > 0) {
	return true;
}
```

Reply:
I've tried this way, but the formatsource failed, it report this:

> Create a new variable for the left hand side operand of the '>' operator for better readability:

Reply:
Alright.